tools/pipelines/test-reliability.yml

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# test-reliability pipeline (for the purpose of detecting flaky tests)
+
+name: $(Build.BuildId)
+
+trigger: none
+pr: none
+
+schedules:
+  - cron: "00 03 * * SUN"
+    displayName: Scheduled Tests on Saturday Nights Weekly
+    branches:
+      include:
+      - main
+      - next
+    always: true
+
+variables:
+- group: prague-key-vault
+- name: testWorkspace
+  value: $(Pipeline.Workspace)/test
+
+stages:
+  - ${{ each stageName in parameters.stageNames }}:
+    - stage:
+      displayName: ${{ stageName }}
+      dependsOn: []
+      jobs:
+      - template: templates/include-test-reliability.yml
+        parameters:
+          testPackage: "@fluid-internal/local-server-tests"
+          testWorkspace: ${{ variables.testWorkspace }}
+          testCommand: test:coverage
```

Comment:
Just wondering if `test:coverage` specifically is needed over `test:mocha`? Its definition is `"cross-env FLUID_TEST_COVERAGE=1 npm run test:mocha"` and I couldn't find any other reference to `FLUID_TEST_COVERAGE` in the repo 🤔. Maybe that script needs to be updated in local-server-tests to look like it does in other packages, i.e. `"test:coverage": "nyc npm test -- --reporter xunit --reporter-option output=nyc/junit-report.xml --exit",`